packages/golos-ui/metadata.json

```diff
@@ -0,0 +1 @@
+{"fontId":"golos-ui","fontName":"Golos UI","subsets":["latin-ext"],"weights":[400,500,600],"styles":["normal"],"defSubset":"latin-ext","variable":false,"lastModified":"2022-01-31","version":"v1.1","category":"sans-serif","source":"https://befonts.com/golos-ui-sans-serif-font.html","license":"https://befonts.com/free-for-commercial-use","type":"uniwidth"}
```

Comment:
Oh I'm sorry I didn't catch this in the last review. I'm not sure what `type: "uniwidth"` means, but that key is usually used as an identifier for if it is a Google Font or not. You don't need to rebuild the whole package again, just change the value to "other" and this should get merged!

Reply:
✅  No problem, thanks for your feedback :)
FYI : 
https://en.wikipedia.org/wiki/Uniwidth_typeface